packages/near-api-js/src/wallet-account.ts

```diff
@@ -102,7 +102,16 @@ export class WalletConnection {
             });
         }
         this._near = near;
-        const authDataKey = appKeyPrefix + LOCAL_STORAGE_KEY_SUFFIX;
+        const DEPRECATED_authDataKey = appKeyPrefix + LOCAL_STORAGE_KEY_SUFFIX;
+        const DEPRECATED_authData = JSON.parse(window.localStorage.getItem(DEPRECATED_authDataKey));
```

Comment:
Doesn't throw on my side since `window.localStorage.getItem(DEPRECATED_authDataKey)` is `null` and `JSON.parse(null)` is fine but we should have a fallback here or a catch block still imo

Reply:
Actually, it seems like this will be fine, per https://html.spec.whatwg.org/multipage/webstorage.html#dom-storage-getitem since getItem will return `null` rather than `undefined` :+1:

Reply:
Thanks, since it's spec we'll leave as is 🙂